.codecov.yml

```diff
@@ -0,0 +1,5 @@
+ignore: 
+  - "examples/" # just example usage, not part of library to be tested
+  - "atlasdb-ete-tests/" # executed elsewhere
```

Comment:
What do you mean by `# executed elsewhere`? I'm assuming you mean something like `# these run in docker containers so they're not tracked`